pvlib/pvsystem.py

```diff
@@ -170,12 +170,12 @@ class PVSystem(object):
     def __init__(self,
                  surface_tilt=0, surface_azimuth=180,
                  albedo=None, surface_type=None,
-                 module=None, module_type='glass_polymer',
+                 module=None, module_type=None,
```

Comment:
I think changing the default to None should be mentioned in whatsnew

Reply:
good catch. added a few awkward sentences to `API Changes with Deprecations`